src/backend/setup.cfg

```diff
@@ -1,13 +1,15 @@
 [metadata]
 name = marsha
 description = A FUN video provider for Open edX
-long_description = file:README.rst
+long_description = file:README.md
```

Comment:
This file does not exist. As we won't distribute this package, you can remove this line.